sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/util/common/Counter.java

```diff
@@ -207,10 +243,20 @@
   public abstract CounterMean<T> getAndResetMeanDelta();
 
   /**
+   * Returns the counter's flat name.
+   */
+  public String getFlatName() {
+    return name.getFlatName();
+  }
+
+  /**
    * Returns the counter's name.
+   *
+   * @deprecated use {@link #getFlatName}.
    */
+  @Deprecated
```

Comment:
Do we need to `@Deprecated` this? Could we not just have `getName` return the `CounterNameAndMetadata`?